### ISSUE TITLE
BUG UPDATE: SQLSTATE[42S01]: Base table or view already exists: 1050 Table 'point_groups' already exists

### ISSUE THREAD

**Issue by TonnyAthylla** [mautic/mautic#14196]
### Mautic Series

5.1.x series

### Mautic installed version

5.1.1

### Way of installing

Other (please specify if relevant in the bug report)

### PHP version

8.1

### What browsers are you seeing the problem on?

Not relevant

### What happened?

I tried to update Mautic, which was in version 5.1.0, to the latest version 5.1.1. And it simply displays a message during the database update.

" An exception occurred while executing a query: SQLSTATE[42S01]: Base table or view already exists: 1050 Table 'REDACTED_point_groups' already exists"

PHP: 8.2
Database: 10.6.19-MariaDB-cll-lve

### How can we reproduce this issue?

I am managing my Mautic through Cpanel.

I had initially installed it through Softaculous and everything was going well, until the new update "5.1.1" came out and I was using "5.1.0".

I followed the steps using the SSH terminal:

1 - I cleared the cache: "php bin/console cache:clear"

2 - I found the update: "php bin/console mautic:update:find"

3 - I ran the update: "php bin/console mautic:update:apply" - (I confirmed with "y")

4 - The terminal itself suggested that I finish the installation with the following command "php bin/console mautic:update:apply --finish" - (the following message was displayed "An error occurred while updating the database. Check the log for more details.

5 - After that, I viewed the logs where it is being sent in this topic.

6 - I applied the following command "php bin/console doctrine:migration:status" and the following information was shown to me:

![MIGRATION_STATUS](https://github.com/user-attachments/assets/bbf31c1c-9681-4902-945b-6a3abfb85824)

7 - I applied the following Command: "php bin/console doctrine:migration:migrate", It displayed this information containing errors:

WARNING! You are about to execute a migration in database "REDACTED" that could result in schema changes and data loss. Are you sure youwish to continue? (yes/no) [yes]:
 > y

[notice] Migrating up to Mautic\Migrations\Versionzz20230929183000
[notice] Migration Mautic\Migrations\Version20190326190241 skipped during Pre-Checks. Reason: "Schema includes this migration"
[notice] Migration Mautic\Migrations\Version20190410143658 skipped during Pre-Checks. Reason: "Schema includes this migration"
[notice] Schema includes this migration
[notice] Migration Mautic\Migrations\Version20190524124819 skipped during Pre-Checks. Reason: "Schema includes this migration"
[notice] Migration Mautic\Migrations\Version20200415135706 skipped during Pre-Checks. Reason: "Schema includes this migration"
[notice] Migration Mautic\Migrations\Version20200513162918 skipped during Pre-Checks. Reason: "The body_text column has already been added to the REDACTED_email_copies table."
[notice] Migration Mautic\Migrations\Version20201026101117 skipped during Pre-Checks. Reason: "Schema includes this migration"
[notice] Migration Mautic\Migrations\Version20210217115150 skipped during Pre-Checks. Reason: "Deleted column already added in tables"
[notice] Index REDACTED_lead_list_alias already exists
[notice] Migration Mautic\Migrations\Version20210420113309 skipped during Pre-Checks. Reason: "Schema includes this migration"
[notice] Index REDACTED_idx_object_field_order_is_published already exists
[notice] Migration Mautic\Migrations\Version20211026152443 skipped during Pre-Checks. Reason: "Schema includes this migration"
[notice] Index REDACTED_idx_frequency_date_added already exists
[notice] Migration Mautic\Migrations\Version20211026153057 skipped during Pre-Checks. Reason: "Schema includes this migration"
[notice] Column already in BOOLEAN type
[notice] Migration Mautic\Migrations\Version20211110070503 skipped during Pre-Checks. Reason: "Schema includes this migration"
[notice] Table REDACTED_contact_export_scheduler already exists
[notice] Migration Mautic\Migrations\Version20220429091934 skipped during Pre-Checks. Reason: "Schema includes this migration"
[notice] Migration Mautic\Migrations\Version20220722074516 skipped during Pre-Checks. Reason: "The deduplicate column has already been added tothe REDACTED_notifications table."
[notice] Migration Mautic\Migrations\Version20221014061125 skipped during Pre-Checks. Reason: "Index REDACTED_webhook_id_date_added already exists"
[notice] Migration Mautic\Migrations\Version20221014081742 skipped during Pre-Checks. Reason: "Schema includes this migration"
[notice] Migration Mautic\Migrations\Version20221128145933 skipped during Pre-Checks. Reason: "Schema includes this migration"
[notice] Migration Mautic\Migrations\Version20230131133732 skipped during Pre-Checks. Reason: "The last_built_time column has already been added to the REDATED_lead_lists table."
[notice] Column REDACTED_asset_downloads.utm_campaign already exists
[notice] Column REDACTED_asset_downloads.utm_content already exists
[notice] Column REDACTED_asset_downloads.utm_medium already exists
[notice] Column REDACTED_asset_downloads.utm_source already exists
[notice] Column REDACTED_asset_downloads.utm_term already exists
[notice] Migration Mautic\Migrations\Version20230321133733 skipped during Pre-Checks. Reason: "Schema includes this migration"
[notice] Migration Mautic\Migrations\Version20230424083829 skipped during Pre-Checks. Reason: "Schema includes this migration"
[notice] Table brmil_plugin_crm_pipedrive_owners was already removed
[notice] Migration Mautic\Migrations\Version20230519154448 skipped during Pre-Checks. Reason: "Schema includes this migration"
[notice] Migration is not required.
[notice] Migration Mautic\Migrations\Version20230606111852 skipped during Pre-Checks. Reason: "Schema includes this migration"
[notice] Table REDACTED_point_groups already exists
[notice] Table REDACTED_point_group_contact_score already exists
[notice] Column REDACTED_points.group_id already exists
[notice] Column REDACTED_point_triggers.group_id already exists
[notice] Column REDACTED_lead_points_change_log.group_id already exists
[error] Migration Mautic\Migrations\Version20230621074925 failed during Execution. Error: "An exception occurred while executing a query: SQLSTATE[42S01]: Base table or view already exists: 1050 Table 'REDACTED_point_groups' already exists"

In ExceptionConverter.php line 45:

  An exception occurred while executing a query: SQLSTATE[42S01]: Base table or view already exists: 1050 Table 'REDACTED_point_groups' already
  exists

In Exception.php line 28:

  SQLSTATE[42S01]: Base table or view already exists: 1050 Table 'REDACTED_point_groups' already exists

In Connection.php line 71:

  SQLSTATE[42S01]: Base table or view already exists: 1050 Table 'REDACTED_point_groups' already exists

doctrine:migrations:migrate [--write-sql [WRITE-SQL]] [--dry-run] [--query-time] [--allow-no-migration] [--all-or-nothing [ALL-OR-NOTHING]] [--configuration CONFIGURATION] [--em EM] [--conn CONN] [--] [<version>]

### Relevant log output

```shell
[2024-10-14T08:56:12.610823-03:00] mautic.ERROR: Doctrine\DBAL\Exception\TableExistsException: An exception occurred while executing a query: SQLSTATE[42S01]: Base table or view already exists: 1050 Table 'REDACTED_point_groups' already exists (uncaught exception) at /home/REDACTED/public_html/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php line 45 while running console command `doctrine:migrations:migrate` [stack trace] #0 /home/REDACTED/public_html/vendor/doctrine/dbal/src/Connection.php(1938): Doctrine\DBAL\Driver\API\MySQL\ExceptionConverter->convert(Object(Doctrine\DBAL\Driver\PDO\Exception), Object(Doctrine\DBAL\Query)) #1 /home/REDACTED/public_html/vendor/doctrine/dbal/src/Connection.php(1880): Doctrine\DBAL\Connection->handleDriverException(Object(Doctrine\DBAL\Driver\PDO\Exception), Object(Doctrine\DBAL\Query)) #2 /home/REDACTED/public_html/vendor/doctrine/dbal/src/Connection.php(1105): Doctrine\DBAL\Connection->convertExceptionDuringQuery(Object(Doctrine\DBAL\Driver\PDO\Exception), 'CREATE TABLE `b...', Array, Array) #3 /home/REDACTED/public_html/vendor/doctrine/migrations/lib/Doctrine/Migrations/Version/DbalExecutor.php(298): Doctrine\DBAL\Connection->executeQuery('CREATE TABLE `b...', Array, Array) #4 /home/REDACTED/public_html/vendor/doctrine/migrations/lib/Doctrine/Migrations/Version/DbalExecutor.php(173): Doctrine\Migrations\Version\DbalExecutor->executeResult(Object(Doctrine\Migrations\MigratorConfiguration)) #5 /home/REDACTED/public_html/vendor/doctrine/migrations/lib/Doctrine/Migrations/Version/DbalExecutor.php(95): Doctrine\Migrations\Version\DbalExecutor->executeMigration(Object(Doctrine\Migrations\Metadata\MigrationPlan), Object(Doctrine\Migrations\Version\ExecutionResult), Object(Doctrine\Migrations\MigratorConfiguration)) #6 /home/REDACTED/public_html/vendor/doctrine/migrations/lib/Doctrine/Migrations/DbalMigrator.php(107): Doctrine\Migrations\Version\DbalExecutor->execute(Object(Doctrine\Migrations\Metadata\MigrationPlan), Object(Doctrine\Migrations\MigratorConfiguration)) #7 /home/REDACTED/public_html/vendor/doctrine/migrations/lib/Doctrine/Migrations/DbalMigrator.php(71): Doctrine\Migrations\DbalMigrator->executePlan(Object(Doctrine\Migrations\Metadata\MigrationPlanList), Object(Doctrine\Migrations\MigratorConfiguration)) #8 /home/REDACTED/public_html/vendor/doctrine/migrations/lib/Doctrine/Migrations/DbalMigrator.php(157): Doctrine\Migrations\DbalMigrator->executeMigrations(Object(Doctrine\Migrations\Metadata\MigrationPlanList), Object(Doctrine\Migrations\MigratorConfiguration)) #9 /home/REDACTED/public_html/vendor/doctrine/migrations/lib/Doctrine/Migrations/Tools/Console/Command/MigrateCommand.php(211): Doctrine\Migrations\DbalMigrator->migrate(Object(Doctrine\Migrations\Metadata\MigrationPlanList), Object(Doctrine\Migrations\MigratorConfiguration)) #10 /home/REDACTED/public_html/vendor/symfony/console/Command/Command.php(298): Doctrine\Migrations\Tools\Console\Command\MigrateCommand->execute(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\NullOutput)) #11 /home/REDACTED/public_html/vendor/symfony/console/Application.php(1058): Symfony\Component\Console\Command\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\NullOutput)) #12 /home/REDACTED/public_html/vendor/symfony/framework-bundle/Console/Application.php(96): Symfony\Component\Console\Application->doRunCommand(Object(Doctrine\Migrations\Tools\Console\Command\MigrateCommand), Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\NullOutput)) #13 /home/REDACTED/public_html/vendor/symfony/console/Application.php(301): Symfony\Bundle\FrameworkBundle\Console\Application->doRunCommand(Object(Doctrine\Migrations\Tools\Console\Command\MigrateCommand), Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\NullOutput)) #14 /home/REDACTED/public_html/vendor/symfony/framework-bundle/Console/Application.php(82): Symfony\Component\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\NullOutput)) #15 /home/REDACTED/public_html/vendor/symfony/console/Application.php(171): Symfony\Bundle\FrameworkBundle\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\NullOutput)) #16 /home/REDACTED/public_html/app/bundles/CoreBundle/Update/Step/UpdateSchemaStep.php(51): Symfony\Component\Console\Application->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\NullOutput)) #17 /home/REDACTED/public_html/app/bundles/CoreBundle/Command/ApplyUpdatesCommand.php(137): Mautic\CoreBundle\Update\Step\UpdateSchemaStep->execute(Object(Symfony\Component\Console\Helper\ProgressBar), Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput)) #18 /home/REDACTED/public_html/app/bundles/CoreBundle/Command/ApplyUpdatesCommand.php(99): Mautic\CoreBundle\Command\ApplyUpdatesCommand->finishUpgrade(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput), Object(Symfony\Component\Console\Helper\ProgressBar)) #19 /home/REDACTED/public_html/vendor/symfony/console/Command/Command.php(298): Mautic\CoreBundle\Command\ApplyUpdatesCommand->execute(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput)) #20 /home/REDACTED/public_html/vendor/symfony/console/Application.php(1058): Symfony\Component\Console\Command\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput)) #21 /home/REDACTED/public_html/vendor/symfony/framework-bundle/Console/Application.php(96): Symfony\Component\Console\Application->doRunCommand(Object(Mautic\CoreBundle\Command\ApplyUpdatesCommand), Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput)) #22 /home/REDACTED/public_html/vendor/symfony/console/Application.php(301): Symfony\Bundle\FrameworkBundle\Console\Application->doRunCommand(Object(Mautic\CoreBundle\Command\ApplyUpdatesCommand), Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput)) #23 /home/REDACTED/public_html/vendor/symfony/framework-bundle/Console/Application.php(82): Symfony\Component\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput)) #24 /home/REDACTED/public_html/vendor/symfony/console/Application.php(171): Symfony\Bundle\FrameworkBundle\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput)) #25 /home/REDACTED/public_html/bin/console(16): Symfony\Component\Console\Application->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput)) #26 {main} [] {"hostname":"pro119.dnspro.com.br","pid":225758}
[2024-10-14T08:56:12.613537-03:00] mautic.WARNING: Command `doctrine:migrations:migrate` exited with status code 255 [] {"hostname":"pro119.dnspro.com.br","pid":225758}
[2024-10-14T08:56:12.643550-03:00] mautic.WARNING: Command `mautic:update:apply` exited with status code 1 [] {"hostname":"pro119.dnspro.com.br","pid":225758}
[2024-10-14T08:57:48.213966-03:00] mautic.ERROR: Doctrine\DBAL\Exception\TableExistsException: An exception occurred while executing a query: SQLSTATE[42S01]: Base table or view already exists: 1050 Table 'REDACTED_point_groups' already exists (uncaught exception) at /home/REDACTED/public_html/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php line 45 while running console command `doctrine:migrations:migrate` [stack trace] #0 /home/REDACTED/public_html/vendor/doctrine/dbal/src/Connection.php(1938): Doctrine\DBAL\Driver\API\MySQL\ExceptionConverter->convert(Object(Doctrine\DBAL\Driver\PDO\Exception), Object(Doctrine\DBAL\Query)) #1 /home/REDACTED/public_html/vendor/doctrine/dbal/src/Connection.php(1880): Doctrine\DBAL\Connection->handleDriverException(Object(Doctrine\DBAL\Driver\PDO\Exception), Object(Doctrine\DBAL\Query)) #

2 /home/REDACTED/public_html/vendor/doctrine/dbal/src/Connection.php(1105): Doctrine\DBAL\Connection->convertExceptionDuringQuery(Object(Doctrine\DBAL\Driver\PDO\Exception), 'CREATE TABLE `b...', Array, Array) #3 /home/REDACTED/public_html/vendor/doctrine/migrations/lib/Doctrine/Migrations/Version/DbalExecutor.php(298): Doctrine\DBAL\Connection->executeQuery('CREATE TABLE `b...', Array, Array) #4 /home/REDACTED/public_html/vendor/doctrine/migrations/lib/Doctrine/Migrations/Version/DbalExecutor.php(173): Doctrine\Migrations\Version\DbalExecutor->executeResult(Object(Doctrine\Migrations\MigratorConfiguration)) #5 /home/REDACTED/public_html/vendor/doctrine/migrations/lib/Doctrine/Migrations/Version/DbalExecutor.php(95): Doctrine\Migrations\Version\DbalExecutor->executeMigration(Object(Doctrine\Migrations\Metadata\MigrationPlan), Object(Doctrine\Migrations\Version\ExecutionResult), Object(Doctrine\Migrations\MigratorConfiguration)) #6 /home/REDACTED/public_html/vendor/doctrine/migrations/lib/Doctrine/Migrations/DbalMigrator.php(107): Doctrine\Migrations\Version\DbalExecutor->execute(Object(Doctrine\Migrations\Metadata\MigrationPlan), Object(Doctrine\Migrations\MigratorConfiguration)) #7 /home/REDACTED/public_html/vendor/doctrine/migrations/lib/Doctrine/Migrations/DbalMigrator.php(71): Doctrine\Migrations\DbalMigrator->executePlan(Object(Doctrine\Migrations\Metadata\MigrationPlanList), Object(Doctrine\Migrations\MigratorConfiguration)) #8 /home/REDACTED/public_html/vendor/doctrine/migrations/lib/Doctrine/Migrations/DbalMigrator.php(157): Doctrine\Migrations\DbalMigrator->executeMigrations(Object(Doctrine\Migrations\Metadata\MigrationPlanList), Object(Doctrine\Migrations\MigratorConfiguration)) #9 /home/REDACTED/public_html/vendor/doctrine/migrations/lib/Doctrine/Migrations/Tools/Console/Command/MigrateCommand.php(211): Doctrine\Migrations\DbalMigrator->migrate(Object(Doctrine\Migrations\Metadata\MigrationPlanList), Object(Doctrine\Migrations\MigratorConfiguration)) #10 /home/REDACTED/public_html/vendor/symfony/console/Command/Command.php(298): Doctrine\Migrations\Tools\Console\Command\MigrateCommand->execute(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\NullOutput)) #11 /home/REDACTED/public_html/vendor/symfony/console/Application.php(1058): Symfony\Component\Console\Command\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\NullOutput)) #12 /home/REDACTED/public_html/vendor/symfony/framework-bundle/Console/Application.php(96): Symfony\Component\Console\Application->doRunCommand(Object(Doctrine\Migrations\Tools\Console\Command\MigrateCommand), Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\NullOutput)) #13 /home/REDACTED/public_html/vendor/symfony/console/Application.php(301): Symfony\Bundle\FrameworkBundle\Console\Application->doRunCommand(Object(Doctrine\Migrations\Tools\Console\Command\MigrateCommand), Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\NullOutput)) #14 /home/REDACTED/public_html/vendor/symfony/framework-bundle/Console/Application.php(82): Symfony\Component\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\NullOutput)) #15 /home/REDACTED/public_html/vendor/symfony/console/Application.php(171): Symfony\Bundle\FrameworkBundle\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\NullOutput)) #16 /home/REDACTED/public_html/bin/console(16): Symfony\Component\Console\Application->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\NullOutput)) #17 {main} [] {"hostname":"pro119.dnspro.com.br","pid":226160}
[2024-10-14T08:57:48.216896-03:00] mautic.WARNING: Command `doctrine:migrations:migrate` exited with status code 255 [] {"hostname":"pro119.dnspro.com.br","pid":226160}
[2024-10-14T08:57:48.242111-03:00] mautic.WARNING: Command `mautic:update:apply` exited with status code 1 [] {"hostname":"pro119.dnspro.com.br","pid":226160}
```

### Code of Conduct

- [X] I confirm that I have read and agree to follow this project's Code of Conduct

<br /><hr>
Care about this issue? Want to get it resolved sooner? If you are a <a href='https://www.mautic.org/become-a-member-of-mautic'>member of Mautic</a>, you can add some funds to the <a href='https://opencollective.com/mautic/projects/bounties'>Bounties Project</a> so that the person who completes this task can claim those funds once it is merged by a member of the core team! Read the docs <a href='https://contribute.mautic.org/product-team/mautic-bounty-programme'>here.</a>

**Comment by TonnyAthylla** [comment]
Mautic seems to be working, but when I apply it to the URL "https://my-website.com/s/update/schema", it displays the following message:

"An error occurred while updating the database. Check the log for more details."
![error_mautic](https://github.com/user-attachments/assets/32fbe637-4645-46a5-94d6-039e2d428b8f)

**Comment by RCheesley** [comment]
Thanks for reporting the bug @TonnyAthylla however this is a duplicate of several issues already reported. Please review this issue which has a detailed run down of the issue and what has been done to resolve it: https://github.com/mautic/mautic/issues/12993.
